tools/StaticAnalysis/Exceptions/Az.Storage/MissingAssemblies.csv

```diff
@@ -2,4 +2,14 @@
-"Az.Storage","Mono.Posix.NETStandard","1.0.0.0","Microsoft.Azure.Storage.DataMovement","0","3000","Missing assembly Mono.Posix.NETStandard referenced from Microsoft.Azure.Storage.DataMovement","Ensure that the assembly is included in the Wix file or directory"
\ No newline at end of file
+"Az.Storage","Mono.Posix.NETStandard","1.0.0.0","Microsoft.Azure.Storage.DataMovement","0","3000","Missing assembly Mono.Posix.NETStandard referenced from Microsoft.Azure.Storage.DataMovement","Ensure that the assembly is included in the Wix file or directory"
+"Az.Storage","System.Buffers","4.0.2.0","Azure.Core","0","3000","Missing assembly System.Buffers referenced from Azure.Core","Ensure that the assembly is included in the Wix file or directory"
+"Az.Storage","System.Buffers","4.0.2.0","Azure.Storage.Blobs","0","3000","Missing assembly System.Buffers referenced from Azure.Storage.Blobs","Ensure that the assembly is included in the Wix file or directory"
+"Az.Storage","System.Buffers","4.0.2.0","Azure.Storage.Common","0","3000","Missing assembly System.Buffers referenced from Azure.Storage.Common","Ensure that the assembly is included in the Wix file or directory"
+"Az.Storage","System.Buffers","4.0.2.0","Azure.Storage.Files.DataLake","0","3000","Missing assembly System.Buffers referenced from Azure.Storage.Files.DataLake","Ensure that the assembly is included in the Wix file or directory"
+"Az.Storage","System.Text.Encodings.Web","4.0.4.0","System.Text.Json","0","3000","Missing assembly System.Text.Encodings.Web referenced from System.Text.Json","Ensure that the assembly is included in the Wix file or directory"
+"Az.Storage","System.Numerics.Vectors","4.1.3.0","System.Text.Json","0","3000","Missing assembly System.Numerics.Vectors referenced from System.Text.Json","Ensure that the assembly is included in the Wix file or directory"
+"Az.Storage","System.Buffers","4.0.2.0","System.Text.Json","0","3000","Missing assembly System.Buffers referenced from System.Text.Json","Ensure that the assembly is included in the Wix file or directory"
+"Az.Storage","System.Runtime.CompilerServices.Unsafe","4.0.5.0","System.Text.Json","0","3000","Missing assembly System.Runtime.CompilerServices.Unsafe referenced from System.Text.Json","Ensure that the assembly is included in the Wix file or directory"
+"Az.Storage","System.Text.Encodings.Web","4.0.4.0","System.Text.Json","0","3000","Missing assembly System.Text.Encodings.Web referenced from System.Text.Json","Ensure that the assembly is included in the Wix file or directory"
+"Az.Storage","System.Runtime.CompilerServices.Unsafe","4.0.5.0","System.Text.Json","0","3000","Missing assembly System.Runtime.CompilerServices.Unsafe referenced from System.Text.Json","Ensure that the assembly is included in the Wix file or directory"
```

Comment:
Ideally we should not add these items here, I'm double checking locally.

Reply:
These are added by Mark in https://github.com/wastoresh/azure-powershell/pull/5

Reply:
Please change:
```suggestion
 "Az.Storage","System.Text.Encodings.Web","4.0.4.0","System.Text.Json","0","3000","Missing assembly System.Text.Encodings.Web referenced from System.Text.Json","Ensure that the assembly is included in the Wix file or directory"
  "Az.Storage","System.Numerics.Vectors","4.1.3.0","System.Text.Json","0","3000","Missing assembly System.Numerics.Vectors referenced from System.Text.Json","Ensure that the assembly is included in the Wix file or directory"
  "Az.Storage","System.Runtime.CompilerServices.Unsafe","4.0.5.0","System.Text.Json","0","3000","Missing assembly System.Runtime.CompilerServices.Unsafe referenced from System.Text.Json","Ensure that the assembly is included in the Wix file or directory"
```